__tests__/app.test.js

```diff
@@ -13,10 +12,22 @@ describe("GET /api", () => {
     return request(app)
       .get("/api")
       .expect(200)
-      .then(({ body }) => {
-        expect(body).toEqual(endpoints);
-        expect(endpoints).toHaveProperty("GET /api/topics");
-        expect(endpoints).toHaveProperty("GET /api");
+      .then(({ body: { endpoints } }) => {
+        for (const endpoint in endpoints) {
+          expect(endpoints[endpoint]).toHaveProperty("description");
+          //not all endpoints has the below two, might compare actual JSON file instead...
```

Comment:
yes! comparing to the JSON file is absolutely fine and will make future tests much easier as you will not have to amend for every endpoint.